packages/mesos/buildinfo.json

```diff
@@ -3,8 +3,8 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/mesosphere/mesos",
-    "ref": "a22b5dd552168a8dd925682f3b349059e10fc3d3",
-    "ref_origin" : "dcos-mesos-1.0.1"
+    "ref": "0950e8664d1b181e82dabe958d9cc1d43a5d7612",
+    "ref_origin" : "dcos-mesos-1.0.x-7b322c9"
```

Comment:
The chance of an official 1.0.2 is unlikely, so I still like this strategy of using the WIP 1.0.x branch at this SHA, which will eventually become 1.0.2 (after a few more unrelated fixes land), rather than cherry-pick just the 2 we need for executor SSL environment on top of 1.0.1.
TL;DR: We should use this branch and SHA in 1.8.5.